src/editor/components/modals/file-modal/title.jsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
```

Comment:
Is this `title.jsx` element not something that we can generalize? We have other modals that have the same style. Any chance we can have one component in all places? I'm sure at this point we're not using one title component for all the other cases, but we probably should!